src/utils/rest-api.js

```diff
@@ -567,14 +567,19 @@ export function duplicateContingencyList(
 export function getContingencyList(type, id) {
     let url = PREFIX_ACTIONS_QUERIES;
```

Comment:
When does this initialisation is still used ?
Chances are (I've not read the specifications) that one could factorize out
that `PREFIX_EXPLORE_SERVER_QUERIES + /v1/explore/` here and use it in each _rung_ of the if _ladder_